src/current/_includes/releases/v24.3/v24.3.0-alpha.2.md

```diff
@@ -0,0 +1,88 @@
+## v24.3.0-alpha.2
+
+Release Date: October 14, 2024
+
+{% include releases/new-release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v24-3-0-alpha-2-security-updates">Security updates</h3>
+
+- The parameters for an [HBA config entry]({% link v24.3/security-reference/authentication.md %}#hba-configuration-syntax) for LDAP are now validated when the entry is created or amended, in addition to the validation that happens during an authentication attempt. [#132086][#132086]
```

Comment:
@souravcrl PTAL

Reply:
Can you share if there was a note for LDAP release in 24.2 @mdlinville? We will need to add documentation for LDAP release and the feature wiki also since it was in private preview earlier in 24.2. cc: @pritesh-lahoti 
Additionally the validation during authentication attempt, clients may want to look into setting the cluster setting `server.auth_log.sql_sessions.enabled` to true, which provides exact failure LDAP authentication failure reason.

Reply:
This is just release notes for the alpha.2. There should be some v24.2 and v24.3 product change issues automatically created in JIRA, which will track the docs work for the LDAP auth changes.

Reply:
This change has no release note yet in v24.2 -- which v24.2 patch did you expect to see it in? https://www.cockroachlabs.com/docs/releases/v24.2

Reply:
This is the [backport PR ](https://github.com/cockroachdb/cockroach/pull/127124) which got merged into 24.2.0-rc and I also see the changes [here](https://github.com/cockroachdb/cockroach/blob/blathers/backport-release-24.2.0-rc-128178/pkg/ccl/ldapccl/settings.go). I feel somehow this got missed in the last rlease note then if the release note texts are automatically fetched and we may need to add them. Is there a reason why this was excluded?

Reply:
Please add ldap authentication to Feature Availability page for v24.2 as that will help provide context for newer feature work/improvements in 24.3 as per our discussion. Thanks for all the help.